pyaedt/circuit.py

```diff
@@ -360,6 +361,25 @@ def create_schematic_from_netlist(self, file_to_import):
         self.logger.info("Netlist correctly imported into %s", self.design_name)
         return True
 
+    @aedt_exception_handler
+    def read_ibis(self, path):
+        """Create an IBIS model based on the data contained into an *.ibs file.
+
+        Parameters
+        ----------
+        path : str
+            Path of the ibis file.
+
+        Returns
+        ----------
+        :class:`pyaedt.generic.ibis_reader.Ibis`
+            Ibis object exposing all data from the ibis file.
+        """
+
+        reader = ibis_reader.IbisReader()
+        ibis = reader.read_project(path, self)
+        return ibis
```

Comment:
There's no reason to keep around a single use parameter.
```suggestion
        return reader.read_project(path, self)
```